### PR TITLE
Album art

### LINKED ISSUE #47: Album art
**Is your feature request related to a problem? Please describe.**
Album art for songs in search results.

**Describe the solution you'd like**
Use CoverPy library to show album art.

### PR DESCRIPTION
**Pull request by abhiroopdutta**
## Description

Added album art, the search result is returned along with title, video_id and album art url.

## Motivation and Context

Fixes #47 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: server/modules/coverpy.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import os
+import requests
+
+class NoResultsException(Exception):
```
Comment (stealthanthrax) [inline server/modules/coverpy.py]: @abhiroopdutta , read about and use "@abstractclassmethod" here

File: server/modules/coverpy.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    pass
+
+class Result:
```
Comment (stealthanthrax) [inline server/modules/coverpy.py]: can you also add some descriptive comments about the class at the starting of the class

File: server/modules/coverpy.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+    def art(self, search_query):
+        try:
+	        result = self.get_cover(search_query, 1)
+	        # Set a size for the artwork (first parameter) and get the result url.
+	        return(result.artwork(300))
+        except NoResultsException:
+	        return('http://dalelyles.com/musicmp3s/no_cover.jpg')
+        except requests.exceptions.HTTPError:
+	        return('http://dalelyles.com/musicmp3s/no_cover.jpg')
```
Comment (stealthanthrax) [inline server/modules/coverpy.py]: add eof here

**Review by sansyrox — CHANGES_REQUESTED** [review]
@abhiroopdutta , good job with the PR, but please look at the inline comments

File: server/modules/coverpy.py (hunk 2 above)
Comment (abhiroopdutta) [reply server/modules/coverpy.py]: This is the CoverPy module that I'm using directly, I don't know how exactly is it working.

**Review by sansyrox — APPROVED** [review]
good job

**Pull request merged**